libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -1050,4 +1067,86 @@ export const buildTigerSpecificFunctions = (
             });
         });
     },
+    getAllCSPDirectives: async (): Promise<Array<ICSPDirective>> => {
+        try {
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.getAllEntitiesCspDirectives({});
+                return result.data?.data || [];
+            });
+        } catch (error) {
+            return [];
```

Comment:
Should we call `throw convertApiError(error);` to throw an error up to the app? I think the SDK should not hide any error at all.

Reply:
It's the same https://github.com/gooddata/gooddata-ui-sdk/blob/9b589cc7118952b6540d753e49a96e7b69cdd655/libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts#L458